### DISCUSSION TITLE
能不能加个下载限速的功能

### DISCUSSION THREAD

**Discussion by ximimangya** [AlistGo/alist#4383] [Ideas]
希望可以加一个下载限速功能，不同用户可以单独设置限速，如果直接在服务器改又会降低访问速度啥的，加上这个功能还可以减轻服务器负担还可以缓解下载器多线程下载所带来的压力，谢谢🙏

**Comment by anwen-anyi** [comment]
估计没希望了🤨挺难做的应该

**Reply by ximimangya** [reply]
服务器：我估计没希望了，挺难活的应该

**Comment by SsuJojo** [comment]
不可能，因为alist有时仅充当代理，为客户提供302重定向
可以用别的第三方工具来限速，

**Comment by SeanHeuc** [comment]
多说一句，我们希望alist 可以在更多的机器上运行。你说的这个功能其实并不会减轻机器的负担，如果要想生效，就一定要开启所有的存储盘的本地代理模式，再加上限速api的使用，cpu会有一定的要求。 虽然说可以做成可选功能，但是感觉有类似需求的用户不多，而且很多软件可以做到更专业的控制，比如软路由系统。
（302重定向的网盘alist是没有什么负载的，完全不用有影响性能的担心）
